docs/api-summary.md

```diff
@@ -23,5 +23,5 @@ The API is secured using an API Key.
 |PUT       |/api/templates/{file}      |Stores a template in the template store          |
 |DELETE    |/api/templates/{file}      |Deletes a template                               |
 |POST      |/api/UpdateBaseTemplates   |Updates base templates (deletes existing data). This should be used only when latest version of templates needs to be pulled.
-|POST      |/api/convert/{srcDataType} |Takes data, and temporary templates as input and outputs FHIR data after applying the templates on the data. The entry-point template is passed base64-encoded in templateBase64 parameter, whereas other overriding templates are passed in the templatesOverrideBase64 parameter.  templatesOverrideBase64 is a base64-encoded json object containing map between the template name and the template content.|
-|POST      |/api/convert/{template}    |Takes data and converts to FHIR using the {template} that is stored on the server.|
+|POST      |/api/convert/{srcDataType} |Takes data, and temporary templates as input and outputs FHIR data after applying the templates on the data. The entry-point template is passed base64-encoded in templateBase64 parameter, whereas other overriding templates are passed in the templatesOverrideBase64 parameter.  templatesOverrideBase64 is a base64-encoded json object containing map between the template name and the template content. You can also set `unusedSegments` and `invalidAccess` parameters in the query string to fetch UI-related segments.|
+|POST      |/api/convert/{srcDataType}/{template}    |Takes data and converts to FHIR using the {template} that is stored on the server. You can also set `unusedSegments` and `invalidAccess` parameters in the query string to fetch UI-related segments.|
```

Comment:
{srcDataType} is part of the template name itself. Therefore, we need not include {srcDataType} here. See the output of {{converterbaseURL}}/api/templates to get the names of all the templates.

Reply:
Got it, I'll remove the extra {srcDataType} path parameter.